packages/sonarwhal/docs/contributor-guide/getting-started/images/cli.svg

```diff
@@ -0,0 +1,2 @@
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
This can be even smaller (61 KB => 57 KB):

```svg
<svg xmlns="http://www.w3.org/2000/svg" style="background-color:#fff" width="1196" height="1827" version="1"><g transform="translate(1 1)"><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 60v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 99l-3-7 3 2 4-2z" pointer-events="none"/><rect width="120" height="60" x="387" fill="#fff" stroke="#000" pointer-events="none" rx="9" ry="9"/><switch transform="translate(396 24)"><foreignObject width="102" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:102px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">sonarwhal http://url</div></div></foreignObject><text x="51" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">sonarwhal http://url</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 156v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 195l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M387 100h120v56H387z" pointer-events="none"/><switch transform="translate(389 115)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Read configuration file</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Read configuration file</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M502 248h39" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M546 248l-7 4 2-4-2-3z" pointer-events="none"/><switch transform="translate(502 234)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No<br/></div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No&lt;br&gt;</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 300v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 339l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(451 302)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M447 196l55 52-55 52-55-52z" pointer-events="none"/><switch transform="translate(394 228)"><foreignObject width="106" height="40" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:106px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Config<br/>found in cwd or home?</div></div></foreignObject><text x="53" y="26" font-family="Helvetica" font-size="12" text-anchor="middle">Config&lt;br&gt;found in cwd or home?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M667 248h34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M706 248l-7 4 2-4-2-3z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M547 208h120v68q-30-22-60 0t-60 0v-56z" pointer-events="none"/><switch transform="translate(549 217)"><foreignObject width="116" height="40" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Show warning to user about running the default configuration</div></div></foreignObject><text x="58" y="26" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M767 278v77H513" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M508 355l7-3-2 3 2 4z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M707 218h120v60H707z" pointer-events="none"/><switch transform="translate(709 235)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Set default configuration</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Set default configuration</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 400v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 439l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M387 340h120v60H387z" pointer-events="none"/><switch transform="translate(407 364)"><foreignObject width="80" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:82px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Validate Config</div></div></foreignObject><text x="40" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Validate Config</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M487 480h154" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M646 480l-7 4 2-4-2-3z" pointer-events="none"/><switch transform="translate(488 466)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 520v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 559l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(451 521)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M447 440l40 40-40 40-40-40z" pointer-events="none"/><switch transform="translate(425 474)"><foreignObject width="44" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:46px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Is valid?</div></div></foreignObject><text x="22" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Is valid?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M727 480h34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M766 480l-7 4 2-4-2-3z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M647 520v-50l80-30v80z" pointer-events="none"/><switch transform="translate(649 446)"><foreignObject width="76" height="68" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:76px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit"><br/><br/>Ask to use default configuration</div></div></foreignObject><text x="38" y="40" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M877 480h70V188H607v14" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M607 207l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(879 466)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 532v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 571l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(826 532)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M822 428l55 52-55 52-55-52z" pointer-events="none"/><switch transform="translate(769 467)"><foreignObject width="106" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:106px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">use default configuration?</div></div></foreignObject><text x="53" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">use default configuration?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 652v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 691l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M782 652v-50l80-30v80z" pointer-events="none"/><switch transform="translate(784 578)"><foreignObject width="76" height="68" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:76px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit"><br/><br/>Ask to create an new configuration<br/></div></div></foreignObject><text x="38" y="40" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 796v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 835l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(825 798)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M877 744h310v1052H513" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M508 1796l7-3-2 3 2 4z" pointer-events="none"/><switch transform="translate(879 729)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M822 692l55 52-55 52-55-52z" pointer-events="none"/><switch transform="translate(769 731)"><foreignObject width="106" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:106px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">create a new configuration?</div></div></foreignObject><text x="53" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">create a new configuration?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 916v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 955l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M782 916v-50l80-30v80z" pointer-events="none"/><switch transform="translate(784 849)"><foreignObject width="76" height="54" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:76px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit"><br/><br/>Ask type of configuration<br/></div></div></foreignObject><text x="38" y="33" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><rect width="120" height="60" x="387" y="1766" fill="#fff" stroke="#000" pointer-events="none" rx="9" ry="9"/><switch transform="translate(437 1790)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:22px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Exit</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Exit</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M782 996h-34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M743 996l7-3-2 3 2 4z" pointer-events="none"/><switch transform="translate(761 982)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1036v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1075l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(825 1039)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M822 956l40 40-40 40-40-40z" pointer-events="none"/><switch transform="translate(790 990)"><foreignObject width="64" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:64px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">predefined?</div></div></foreignObject><text x="32" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">predefined?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M622 996h-44v-78" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M578 913l4 7-4-2-3 2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M622 966h120v60H622z" pointer-events="none"/><switch transform="translate(625 990)"><foreignObject width="114" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:114px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Choose configuration</div></div></foreignObject><text x="57" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Choose configuration</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M887 1138h34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M926 1138l-7 4 2-4-2-3z" pointer-events="none"/><switch transform="translate(889 1123)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1200v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1239l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(826 1202)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M822 1076l65 62-65 62-65-62z" pointer-events="none"/><switch transform="translate(759 1118)"><foreignObject width="126" height="40" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:126px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">At least one<br/>resource of each type (except parsers)</div></div></foreignObject><text x="63" y="26" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M1107 1138h40v643H513" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M508 1781l7-3-2 3 2 4z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M927 1087h180v87q-45-28-90 0-45 27-90 0v-72z" pointer-events="none"/><switch transform="translate(929 1096)"><foreignObject width="176" height="54" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:176px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Couldn&apos;t find any installed {{type}}s. Visit https://www.npmjs.com/search?q=%40sonarwhal%2F{{type}}</div></div></foreignObject><text x="88" y="33" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1300v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1339l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M762 1240h120v60H762z" pointer-events="none"/><switch transform="translate(768 1264)"><foreignObject width="108" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:108px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Choose a connector</div></div></foreignObject><text x="54" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Choose a connector</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1400v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1439l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M762 1340h120v60H762z" pointer-events="none"/><switch transform="translate(773 1364)"><foreignObject width="98" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:100px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Choose formatters</div></div></foreignObject><text x="49" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Choose formatters</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1500v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1539l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M762 1440h120v60H762z" pointer-events="none"/><switch transform="translate(787 1464)"><foreignObject width="70" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:72px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Choose rules</div></div></foreignObject><text x="35" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Choose rules</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M822 1634v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M822 1673l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(827 1635)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M775 1587H557V922" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M557 917l4 7-4-2-3 2z" pointer-events="none"/><switch transform="translate(758 1572)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M822 1540l48 47-48 47-47-47z" pointer-events="none"/><switch transform="translate(777 1567)"><foreignObject width="91" height="40" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:91px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">At least one parser<br/>installed?</div></div></foreignObject><text x="46" y="26" font-family="Helvetica" font-size="12" text-anchor="middle">At least one parser&lt;br&gt;installed?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M762 1704H536V918" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M536 913l4 7-4-2-3 2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M762 1674h120v60H762z" pointer-events="none"/><switch transform="translate(780 1698)"><foreignObject width="84" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:86px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Choose parsers</div></div></foreignObject><text x="42" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Choose parsers</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M557 836V370h-44" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M508 370l7-3-2 3 2 4z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M527 852c0-21 60-21 60 0v48c0 21-60 21-60 0z" pointer-events="none"/><path fill="none" stroke="#000" stroke-miterlimit="10" d="M527 852c0 16 60 16 60 0" pointer-events="none"/><switch transform="translate(529 875)"><foreignObject width="56" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:56px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Store config</div></div></foreignObject><text x="28" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Store config</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 620v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 659l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M387 560h120v60H387z" pointer-events="none"/><switch transform="translate(406 584)"><foreignObject width="82" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:84px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Load resources</div></div></foreignObject><text x="41" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Load resources</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 764v36" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 805l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(449 767)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M392 712H223" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M218 712l7-3-2 3 2 4z" pointer-events="none"/><switch transform="translate(371 697)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M447 660l55 52-55 52-55-52z" pointer-events="none"/><switch transform="translate(414 685)"><foreignObject width="66" height="54" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:66px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Any<br/>dependency<br/>related<br/>error?</div></div></foreignObject><text x="33" y="33" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M392 858H253" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M248 858l7-3-2 3 2 4z" pointer-events="none"/><switch transform="translate(360 844)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 910v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 949l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(451 910)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M447 806l55 52-55 52-55-52z" pointer-events="none"/><switch transform="translate(394 838)"><foreignObject width="106" height="40" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:106px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Everything is installed and compatible?</div></div></foreignObject><text x="53" y="26" font-family="Helvetica" font-size="12" text-anchor="middle">Everything is installed and compatible?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M97 712H7v1099h374" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M386 1811l-7 4 2-4-2-3z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M97 672h120v68q-30-22-60 0t-60 0v-56z" pointer-events="none"/><switch transform="translate(99 674)"><foreignObject width="116" height="54" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Module {{moduleName}} not found when loading {{resourceName}}</div></div></foreignObject><text x="58" y="33" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M187 888v34" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M187 927l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M127 828h120v60H127z" pointer-events="none"/><switch transform="translate(129 845)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Check if sonarwhal is installed locally</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Check if sonarwhal is installed locally</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M147 968H87v74" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M87 1047l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(126 954)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M227 968h60v74" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M287 1047l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(229 954)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M187 928l40 40-40 40-40-40z" pointer-events="none"/><switch transform="translate(149 955)"><foreignObject width="76" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:76px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Is installed locally?</div></div></foreignObject><text x="38" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Is installed locally?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M287 1108v28h-80v27" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M207 1168l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M227 1048h120v60H227z" pointer-events="none"/><switch transform="translate(229 1065)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Install/Update packages globally</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Install/Update packages globally</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M87 1108v28h80v27" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M167 1168l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M27 1048h120v60H27z" pointer-events="none"/><switch transform="translate(29 1065)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Install/Update packages locally</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Install/Update packages locally</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M147 1189H67v607h314" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M386 1796l-7 4 2-4-2-3z" pointer-events="none"/><switch transform="translate(132 1175)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No</div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M227 1189h130V980h24" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M386 980l-7 4 2-4-2-3z" pointer-events="none"/><switch transform="translate(230 1175)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M187 1149l40 40-40 40-40-40z" pointer-events="none"/><switch transform="translate(158 1176)"><foreignObject width="58" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:58px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Installation<br/>ok?<br/></div></div></foreignObject><text x="29" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Installation&lt;br&gt;ok?&lt;br&gt;</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 1010v234" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 1249l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M387 950h120v60H387z" pointer-events="none"/><switch transform="translate(389 967)"><foreignObject width="116" height="26" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Validate rules configuration</div></div></foreignObject><text x="58" y="19" font-family="Helvetica" font-size="12" text-anchor="middle">Validate rules configuration</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M407 1290h-74" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M328 1290l7-3-2 3 2 4z" pointer-events="none"/><switch transform="translate(391 1276)"><foreignObject width="15" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">No<br/></div></div></foreignObject><text x="8" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">No&lt;br&gt;</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 1330v110" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 1445l-3-7 3 2 4-2z" pointer-events="none"/><switch transform="translate(450 1331)"><foreignObject width="20" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;white-space:nowrap;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;background-color:#fff">Yes</div></div></foreignObject><text x="10" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Yes</text></switch><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M447 1250l40 40-40 40-40-40z" pointer-events="none"/><switch transform="translate(425 1284)"><foreignObject width="44" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:46px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Is valid?</div></div></foreignObject><text x="22" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Is valid?</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M207 1290h-80v491h254" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M386 1781l-7 4 2-4-2-3z" pointer-events="none"/><path fill="#fff" stroke="#000" stroke-miterlimit="10" d="M207 1250h120v68q-30-22-60 0t-60 0v-56z" pointer-events="none"/><switch transform="translate(209 1252)"><foreignObject width="116" height="54" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:116px;white-space:normal;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Invalid rule configuration in .sonarwhalrc: {{listOfInvalidRules}}</div></div></foreignObject><text x="58" y="33" font-family="Helvetica" font-size="12" text-anchor="middle">[Not supported by viewer]</text></switch><path fill="none" stroke="#000" stroke-miterlimit="10" d="M447 1506v254" pointer-events="none"/><path stroke="#000" stroke-miterlimit="10" d="M447 1765l-3-7 3 2 4-2z" pointer-events="none"/><path fill="#fff" stroke="#000" d="M387 1446h120v60H387z" pointer-events="none"/><switch transform="translate(425 1470)"><foreignObject width="44" height="12" pointer-events="all" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility" style="overflow:visible"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;font-size:12px;font-family:Helvetica;color:#000;line-height:1.2;vertical-align:top;width:46px;white-space:nowrap;word-wrap:normal;text-align:center"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit">Scan url</div></div></foreignObject><text x="22" y="12" font-family="Helvetica" font-size="12" text-anchor="middle">Scan url</text></switch></g></svg>
```

(I'll change this before merging)